src/sql.rs

```diff
@@ -0,0 +1,53 @@
+use jobs;
+use mysql::{OptsBuilder, Params, Pool};
+use serde_json::{json, Number};
+use std::error::Error;
+use std::io::Result;
+use std::sync::{Arc, Mutex};
+use std::time::Duration;
+
+lazy_static! {
+    static ref POOL: Arc<Mutex<Option<Pool>>> = Arc::new(Mutex::new(None));
+}
+
+// helper functions to prevent uglification
+fn err_to_json(e: &dyn Error) -> String {
```

Comment:
this is probably a big no-no but I don't see a better way to do what I want

ofc what I want could just be a big-time no-no too.